zppy/templates/default.ini

```diff
@@ -337,3 +337,5 @@ ilamb_obs = string(default="")
 land_only = boolean(default=False)
 ts_atm_subsection = string(default="atm_monthly_180x360_aave")
 ts_land_subsection = string(default="land_monthly")
+ts_atm_grid = string(default="180x360_aave")
+ts_land_grid = string(default="180x360_aave")
```

Comment:
It might be nice to add comments above these providing a short explanation of what they are. But I also see none of the ILAMB parameters have that anyway. So up to you.

Could be:
```
# Name of the grid used by the relevant `[ts]` `atm` task
ts_atm_grid = string(default="180x360_aave")
# Name of the grid used by the relevant `[ts]` `lnd` task
ts_land_grid = string(default="180x360_aave")
```